src/GafferSceneUI/LightPositionTool.cpp

```diff
@@ -1042,6 +1069,26 @@ bool LightPositionTool::placeTarget( const LineSegment3f &eventLine )
 			);
 		}
 	}
+	else
+	{
+		if( !distanceHandle->getTarget() )
+		{
+			return false;
+		}
+
+		std::optional<V3f> sceneGadgetNormal = sceneGadget->normalAt( eventLine );
+		if( sceneGadgetNormal )
+		{
+			V3f worldNormal;
+			sceneGadget->fullTransform().inverse().transpose().multDirMatrix( sceneGadgetNormal.value(), worldNormal );
+
+			positionAlongNormal(
+				distanceHandle->getTarget().value() * sceneToTransformSpaceInverse,
+				worldNormal,
```

Comment:
From @johnhaddon 

Do we need to normalize here? When testing with Arnold I'm getting strange into a strange situation where the light moves further and further away during a drag.

Reply:
Yes, checking the values coming out of `SceneGadget::normalAt()` showed them not always being `1`. I was finding it to be getting closer and closer to the target with Arnold actually.

Either way, it seems like `normalAt()` should ensure it's returning a normalized value. Fixed in https://github.com/GafferHQ/gaffer/pull/5803/commits/a856f810009a24917ae6d32dca3f0fcf97dee841